python/cugraph/cugraph/dask/centrality/betweenness_centrality.py

```diff
@@ -0,0 +1,218 @@
+
+    CuGraph does not currently support the 'endpoints' and 'weight' parameters
+    as seen in the corresponding networkX call.
+
+    Parameters
+    ----------
+    input_graph: cuGraph.Graph
+        The graph can be either directed (Graph(directed=True)) or undirected.
+        Weights in the graph are ignored, the current implementation uses a parallel
+        variation of the Brandes Algorithm (2001) to compute exact or approximate
+        betweenness. If weights are provided in the edgelist, they will not be
+        used.
+    
+    k : int or list or None, optional (default=None)
+        If k is not None, use k node samples to estimate betweenness.  Higher
+        values give better approximation.  If k is a list, use the content
+        of the list for estimation: the list should contain vertex
+        identifiers. If k is None (the default), all the vertices are used
+        to estimate betweenness.  Vertices obtained through sampling or
+        defined as a list will be used as sources for traversals inside the
+        algorithm.
+
+    normalized : bool, optional (default=True)
+        If True normalize the resulting betweenness centrality values
+
+    endpoints : bool, optional (default=False)
+        If true, include the endpoints in the shortest path counts.
+    
+    seed : optional (default=None)
+        if k is specified and k is an integer, use seed to initialize the
+        random number generator.
+        Using None defaults to a hash of process id, time, and hostname
+        If k is either None or list: seed parameter is ignored
+
+    Returns
+    -------
+    betweenness_centrality : dask_cudf.DataFrame
+        GPU distributed data frame containing two dask_cudf.Series of size V:
+        the vertex identifiers and the corresponding betweenness centrality values.
+
+        ddf['vertex'] : dask_cudf.Series
+            Contains the vertex identifiers
+        ddf['betweenness_centrality'] : dask_cudf.Series
+            Contains the betweenness centrality of vertices
+
+    Examples
+    --------
+    >>> import cugraph.dask as dcg
+    >>> import dask_cudf
+    >>> # ... Init a DASK Cluster
+    >>> #    see https://docs.rapids.ai/api/cugraph/stable/dask-cugraph.html
+    >>> # Download dataset from https://github.com/rapidsai/cugraph/datasets/..
+    >>> chunksize = dcg.get_chunksize(datasets_path / "karate.csv")
+    >>> ddf = dask_cudf.read_csv(datasets_path / "karate.csv",
+    ...                          chunksize=chunksize, delimiter=" ",
+    ...                          names=["src", "dst", "value"],
+    ...                          dtype=["int32", "int32", "float32"])
+    >>> dg = cugraph.Graph(directed=True)
+    >>> dg.from_dask_cudf_edgelist(ddf, source='src', destination='dst')
+    >>> pr = dcg.betweenness_centrality(dg)
+
+    """
+
+    # FIXME: Add a warning stating that weights are not supported in this
+    # this implementation
+
+    client = input_graph._client
```

Comment:
Also import the client from distributed.. (`from dask.distributed import wait, default_client`
) 
```suggestion
    client = default_client()
```